buildenv/jenkins/common/build.groovy

```diff
@@ -572,7 +572,21 @@ def build_all() {
                 timeout(time: 5, unit: 'HOURS') {
                     try {
                         // Cleanup in case an old build left anything behind
-                        cleanWs disableDeferredWipeout: true, deleteDirs: true
+                        // cleanWS doesn't always work on Windows #10834
+                        try {
+                                if (SPEC.contains('win')) {
+                                        if (WORKSPACE != null && !WORKSPACE.isEmpty()) {
```

Comment:
Would this ever not be true? Does `if (WORKSPACE)` do the same thing?

Reply:
Likely yes, tbh I copied this from another block of our script...I'll verify that works before pushing up again.